src/configuration/testcafe-configuration.ts

```diff
@@ -234,6 +241,20 @@ export default class TestCafeConfiguration extends Configuration {
         }
     }
 
+    private async _getBrowserInfo (): Promise<BrowserInfoSource[]> {
+        if (!this._options.browsers.value)
+            return [];
+
+        const browsers = Array.isArray(this._options.browsers.value) ? [...this._options.browsers.value] : [this._options.browsers.value];
+
+        const browserInfo = await Promise.all(browsers.map(browser => {
+            return (browser as BrowserInfo)['alias'] && browser ||
+                   browserProviderPool.getBrowserInfo(browser);
```

Comment:
Please try to avoid `as` if you can, since it weakens type safety.
```suggestion
            return browser instanceof BrowserInfo ? browser : browserProviderPool.getBrowserInfo(browser);
```

Reply:
There isn't a problem, because we just check that 'alias' belongs to 'browser'. It means, that we can return 'browser' without 'getBrowserInfo'. Otherwise, we get browser info via the method 'getBrowserInfo'. However, I think, that better way is to check a property 'browserOption'.